packages/inter-protocol/scripts/add-collateral-core.js

```diff
@@ -70,6 +70,18 @@ export const psmGovernanceBuilder = async ({
           psm: publishRef(
             install('../src/psm/psm.js', '../bundles/bundle-psm.js'),
           ),
+          vaults: publishRef(
+            install(
+              '../src/vaultFactory/vaultFactory.js',
+              '../bundles/bundle-vaults.js',
+            ),
+          ),
+          auction: publishRef(
+            install(
+              '../src/auction/auctioneer.js',
+              '../bundles/bundle-auction.js',
```

Comment:
I think our style is to name the bundle after the contract entrypoint, which is independent of how it's being used in a manifest.
```suggestion
              '../bundles/bundle-auctioneer.js',
```

Reply:
done